pages/campaign/index.js

```diff
@@ -0,0 +1,131 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { rem } from '../../styling/typography'
+import withLayout from '../../components/withLayout'
+import { getLocaleFromQuery } from '../../utils/locale'
+import Head from '../../components/Head'
+import PageSection from '../../components/PageSection'
+import CenteredText from '../../components/CenteredText'
+import TeamMember from '../../components/TeamMember'
+import RoundedImage from '../../components/RoundedImage'
+import Markdown from '../../components/Markdown'
+import { fetchCampaignPage } from '../../api/campaign'
+import withQueryParams from '../../components/withQueryParams'
+import DonationForm from '../../components/DonationForm'
+import imagePropTypes from '../../propTypes/image'
+import teamMemmberPropTypes from '../../propTypes/teamMember'
+
+const StyledRoundedImage = RoundedImage.extend`
+  margin-bottom: 1rem;
+  display: block;
+`
+
+const TeamMemberHeadline = styled.div`
```

Comment:
We don't have a heading style for this already? Is it a unique heading?

Reply:
I think it is. If you think it is used anywhere I can switch it.